ruby/true_automation/lib/true_automation/driver/appium_lib.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../client'
+
+module TrueAutomation
+  module Driver
+    class AppiumLib < Appium::Core::Driver
+      def initialize(opts = {})
```

Comment:
I suggest here to use `*args` too

Reply:
`*args` works correct with this  `super`?
![image](https://user-images.githubusercontent.com/923156/228568484-d3d58391-fd47-483a-8ec5-6f354a910361.png)

Reply:
Currently you accept only one argument which is hash if there is no args
`super` without params I think have the same args as current method.
So all you got you send to super

Reply:
Currently I should not send to `super` because `super` wants `opts` argument as it defined . So if I change it to `*args` I should send `*args` to `super` isn't it?

Reply:
@alexander-sysuiev  checked all works fine. In the next version need to add remote appium and record key.